kitsune/questions/static/questions/js/components/AAQApp.jsx

```diff
@@ -1,87 +0,0 @@
-/* globals React:false */
-/* ecmaFeatures jsx: true */
-import QuestionEditStore from '../stores/QuestionEditStore.es6.js';
-import UserAuthStore from '../../../users/js/stores/UserAuthStore.es6.js';
-import ProductSelector from './ProductSelector.jsx';
-import TopicSelector from './TopicSelector.jsx';
-import TitleContentEditor from './TitleContentEditor.jsx';
-import AttachmentField from './AttachmentField.jsx';
-import UserAuth from './UserAuth.jsx';
-import SubmitQuestion from './SubmitQuestion.jsx';
-import TroubleshootingDataStore from '../stores/TroubleshootingDataStore.es6.js';
-import UrlStore from '../../../sumo/js/stores/UrlStore.es6.js';
-import {pathStructure} from '../../../sumo/js/constants/UrlConstants.es6.js';
```

Comment:
UrlConstants is only imported in UrlActions and UrlStore neither of which are imported anywhere else, they can all be removed